src/core_mymalloc.c

```diff
@@ -36,7 +35,7 @@ void *mymalloc(size_t n)
     if(TotMem > HighMarkMem) {
         HighMarkMem = TotMem;
         if(HighMarkMem > OldPrintedHighMark + 10 * 1024.0 * 1024.0) {
-            printf("\nnew high mark = %g MB\n", HighMarkMem / (1024.0 * 1024.0));
+            /* printf("\nnew high mark = %g MB\n", HighMarkMem / (1024.0 * 1024.0)); */
```

Comment:
Should there perhaps be a `-DVERBOSE_MEMORY` flag to enable showing these?  I think they're useful to have access to but I know it messes up the progress bar and adds unnecessary clutter for most users.